manifest.toml

```diff
@@ -2,10 +2,10 @@ packaging_format = 2
 
 id = "aeneria"
 name = "æneria"
-description.en = "Dashboard to analyse your energy consumption data from Linky & weather"
-description.fr = "Tableau de bord pour analyser votre consomation d'énergie à partir des données d'un Linky et de la météo"
+description.en = "Dashboard to analyse your energy consumption data from Linky, Gazpar & weather"
+description.fr = "Dashboard pour analyser vos données de consommation d'énergie Linky, Gazpar et météo"
 
-version = "1.1.6~ynh5"
+version = "2.0.3~ynh1"
```

Comment:
Don't forget to update this (as in READMEs)

Reply:
:+1: